packages/main/src/themes/Button.css

```diff
@@ -106,6 +106,9 @@
 	/* calculating the minimum width by removing the icon size  */
 	min-width: calc(var(--_ui5_button_base_min_width) - var(--_ui5_button_base_icon_margin) - 1rem);
 }
+:host([disabled]:active) {
```

Comment:
can you put a comment here, and improve the PR description that hover should work but click should not work on disabled? this CSS is too decoupled from both the hover/tooltip and the click prevention and it is hard to understand its purpose when looking at this code alone.

Reply:
Sorry, but I saw the comment after they merged. I will release a new PR and reflect the suggestions.